src/frontend/src/styles/main.css

```diff
@@ -1763,6 +1772,11 @@ by all browsers (FF is missing) */
   border-color: red;
 }
 
+/* Use a fake font for "password"-style pin input */
+.c-input--pin__hidden {
```

Comment:
not familiar with the style guide, but wouldn't a a class name `pin__password` be more meaningful that `hidden`? spontaneously I would assume `display: hidden` is used but again, do not know the css here.

Reply:
good catch, called it `__secret`